storage/docs/rest.md

```diff
@@ -1,69 +1,83 @@
 # REST Api
 
-## Store a file
+## URL scheme
+
+The REST Api uses the following scheme for urls
+
+``` /files/<namespace>/<key> ```
+
+The '<namespace>' component cannot include the the '/' character, while the '<key>' can include zero
+or more directories e.g:
+* ``` /files/libero/foo.txt ```
+* ``` /files/libero/baz/bar/foo.txt ```
+
+## Storing a file
+
+To store a file with a namespace of ```libero``` and a key of ```directory/file.pdf```, the following request should be
+used
 
 ```
-PUT /files/namespace/directory/file.ext
+PUT /files/libero/directory/file.pdf
+
```

Comment:
Do we need to send the Content-Type?

Reply:
What happened to `namespace` we need that for multi tenanting ... its essentially `org/journal`

Reply:
see a few lines above:

> To store a file with a namespace of ```libero``` and a key of ```directory/file.pdf```, the following request should be used.

Was thinking that having a more concrete example might be better for illustration

Reply:
Yes.. I understand now ... maybe the abstract one should have been written like:

```
PUT /files/<namespace>/<directory>/<file.ext>
```